pkg/aws/appmesh.go

```diff
@@ -284,6 +285,20 @@ func (c *Cloud) CreateVirtualNode(ctx context.Context, vnode *appmeshv1beta1.Vir
 		}
 	}
 
+	if vnode.Spec.Logging != nil {
+		if vnode.Spec.Logging.AccessLog != nil {
+			if vnode.Spec.Logging.AccessLog.File != nil {
```

Comment:
Nit: I would prefer to join the if-statements, but maybe that's just me?

Reply:
yeah joined them, in future if there are variances then we can revisit breaking them.